subsys/can/canopen/Kconfig

```diff
@@ -0,0 +1,25 @@
+# Kconfig - CAN J1939
+#
+# Copyright (c) 2019 Erwin Rol <erwin@erwinrol.com>
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+menuconfig CANOPEN
+	bool "CANOpen stack"
+	depends on CAN_STACKS
+	help
+	  Enable support of CANOpen stack.
+
+if CANOPEN
+
+module = CANOPEN
+module-str = canopen
+source "subsys/logging/Kconfig.template.log_config"
+
+config CANOPEN_DUMMY
```

Comment:
I think you don't need this since you can use CANOPEN for the rest.

Reply:
"this" ? Are you revering to the CANOPEN_DUMMY?   Because that was just a place holder for some optional/configurable option. 

Or do you mean the menuconfig ? That is a matter of taste I guess, I like it better when things are in their own menu instead one huge list with more and more things.

Reply:
Forget this comment. I was wrong. I thought This would be set in the general can Kconfig.
This is Just fine as it is